Sources/SwiftInspectorVisitors/Tests/PropertyInfoSpec.swift

```diff
@@ -2,3 +2,131 @@
 // Copyright © 2021 Airbnb Inc. All rights reserved.
 
 import Foundation
+import Nimble
+import Quick
+import SwiftInspectorTestHelpers
+import SwiftSyntax
+
+@testable import SwiftInspectorVisitors
+
+final class PropertyInfoParadigmSpec: QuickSpec {
+
+  let undefinedConstantTestCase = PropertyInfo.Paradigm.undefinedConstant
+  let undefinedConstantTestCaseData = """
+    {
+      "caseValue": 0
+    }
+    """.data(using: .utf8)!
+
+  let definedConstantTestCase = PropertyInfo.Paradigm.definedConstant("Foo()")
+  let definedConstantTestCaseData = """
+    {
+      "caseValue": 1,
+      "initializerDescription": "Foo()"
+    }
```

Comment:
I'd recommend creating a helper function that gives you this, that way if we ever need to update this you only do it at one place cc @bachand

Reply:
Since there are a number of different permutations for these keys it's not completely clear to me how to write a simple helper. Can you help me understand what you're thinking? @fdiaz